lib/ffmpeg/qsv/util.py

```diff
@@ -120,6 +120,7 @@ def mapprofile(codec, profile):
     },
     "av1-8"   : {
       "main"      : "main",
+      "Profile0"  : "main",
```

Comment:
Why do we need a new user config mapping to "main"?

Reply:
from vainfo:
      VAProfileAV1Profile0            : VAEntrypointVLD
      VAProfileAV1Profile0            : VAEntrypointEncSliceLP

Maybe we need to remove the old ""main"      : "main"," how do you think?

Reply:
Yes, we can remove other one.  Also, change "Profile0" to "profile0" (lowercase).